docs/src/modules/spring/pages/actions-as-controller.adoc

```diff
@@ -1 +1,112 @@
 = Actions as Controllers
+
+Actions can be used to implement link:https://en.wikipedia.org/wiki/Model%E2%80%93view%E2%80%93controller[MVC Controllers] by
+acting as the external interface of a service, receiving requests, operating over the requests values and forwarding the call
+to other components in the same service.
+
+To illustrate how you can use an Action as a Controller, we will build on top of the xref:structuring-your-application.adoc[Value Entity Shopping Cart] example, adding a new Action to the existing shopping cart service.
+
+== Forwarding Commands
+
+The `forward` effect allows us to transform or further validate an incoming request before passing it on to another
+component and have the response message directly passed back to the client making the request. The response from the
+forwarded operation must have the same response type as the original command.
+
+In this example we accept requests with the same structure as the entity, by receiving a `LineItem`, but add some additional verification of the
+request and only conditionally forward the request to the entity if the verification is successful:
+
+
+.src/main/java/com/example/api/ShoppingCartController.java
+[source,java]
+----
+include::example$spring-valueentity-shopping-cart/src/main/java/com/example/api/ShoppingCartController.java[tag=forward]
```

Comment:
I think it would be nice to explain what is the `ActionCreationContext creationContext` that appears on the constructor.  Especially because `kalixContext` maps to a variable but the `creationContext` doesn't.

Reply:
I think it should actually work without it, I'll try removing

Reply:
good idea